component/otelcol/consumer.go

```diff
@@ -0,0 +1,19 @@
+package otelcol
+
+import (
+	otelconsumer "go.opentelemetry.io/collector/consumer"
+)
+
+// Consumer is a combined OpenTelemetry Collector consumer which can consume
+// any telemetry signal.
+type Consumer interface {
+	otelconsumer.Traces
+	otelconsumer.Metrics
+	otelconsumer.Logs
+}
+
+// ConsumerExports is a common Exports type for Flow components which are
+// otelcol processors or otelcol exporters.
+type ConsumerExports struct {
+	Input Consumer `river:"input,attr"`
```

Comment:
I don't _love_ the name Input here, but we can leave it for now. If I get this correctly, it's like what we used as a `prometheus.Receiver` right?

Reply:
yeah, we could also call it `receiver` if you think that would help (or if that might confuse people since they're not compatible, we could call it `consumer`)

Reply:
Let's leave it as-is for now, just wanted a quick verification.